interfaces/Glitter/templates/include_queue.tmpl

```diff
@@ -78,7 +78,7 @@
         <!-- ko if: !hasQueue() -->
         <tbody class="no-downloads">
             <tr>
-                <td colspan="6">
+                <td colspan="6" data-bind="attr: { 'colspan': 6 + extraQueueColumns().length }">
```

Comment:
I missed that part. Shouldn't this be 5 + extraQueueColumns().length, because the old extra column is removed?

Reply:
Good catched, I fixed it.